DiscordChatExporter.Domain/Discord/DiscordClient.cs

```diff
@@ -171,9 +171,16 @@ public async IAsyncEnumerable<Role> GetGuildRolesAsync(Snowflake guildId)
 
         public async ValueTask<ChannelCategory> GetChannelCategoryAsync(Snowflake channelId)
         {
-            var response = await GetJsonResponseAsync($"channels/{channelId}");
+            try
+            {
+                var response = await GetJsonResponseAsync($"channels/{channelId}");
+                return ChannelCategory.Parse(response);
+            }
+            catch (DiscordChatExporterException exception)
```

Comment:
Please add some comments here (essentially, what you wrote in the PR description) to explain why this is needed.

Reply:
Done!